packages/blocks/src/database-block/components/cell-container.ts

```diff
@@ -17,7 +16,7 @@ export class DatabaseCellContainer
   implements RowHost
 {
   static override styles = css`
-    :host {
+    affine-database-cell-container {
```

Comment:
It appears that the `DatabaseCellElement` is a LitElement, not a ShadowlessElement. Therefore, I believe using `:host` is correct.

Reply:
It's a `ShadowlessElement`.